ws_bulk_report_generator/bulk_report_generator.py

```diff
@@ -44,7 +45,9 @@
 def parse_args():
     parser = argparse.ArgumentParser(description=__description__)
     parser.add_argument('-u', '--userKey', help="WS User Key", dest='ws_user_key', type=str, required=True)
-    parser.add_argument('-k', '--token', help="WS Token", dest='ws_token', type=str, required=True)
+    token_group = parser.add_mutually_exclusive_group(required=True)
+    token_group.add_argument('-k', '--token', help="WS Token", dest='ws_token', type=str)
+    token_group.add_argument('-l', '--list-token', help="WS Token List (Comma Separated)", dest='ws_token_list', type=str)
```

Comment:
A better approach will be to enable a list of tokens under the `-k / --token` argument.
So that if the list of tokens / single token is provided the logic will know how to handle it and from the user perspective it will only require to maintain it under a single argument .

Reply:
This should be fixed with commit hash: b612b88

Reply:
According to conversation with Ram, we decided to add the option for either a flat list, or set it to one sheet per organization. This is reflected in commit hash: dd0cb1a.